katago_server/startposes/models/startpos.py

```diff
@@ -0,0 +1,66 @@
+import os
+import numpy as np
+import random
+
+from django.core.files.storage import FileSystemStorage
+from django.core.validators import RegexValidator
+from django.db.models import (
+    Model,
+    JSONField,
```

Comment:
JsonField must be imported from postgres.

Reply:
Yes, fixed in later commit.